spacy_experimental/cli/benchmark.py

```diff
@@ -0,0 +1,101 @@
+from typing import Iterable, Optional
```

Comment:
This directory needs `__init__.py` or the packaging doesn't work. This needs some kind of test.